var/spack/repos/builtin/packages/py-yt/package.py

```diff
@@ -65,7 +66,12 @@ class PyYt(PythonPackage):
     depends_on("py-sympy", type=('build', 'run'))
     depends_on("python @2.7:2.999,3.4:")
 
-    @run_after('install')
```

Comment:
`develop` is a case which is supported in setuptools, but for which not all projects may be configured to use. This specific package does use it, and I like to use it for my work. However I think the odd package may require the modification of build phases at install-time, however there may be a better solution to this that I'm not aware of.